src/main/kotlin/com/launchdarkly/intellij/LaunchDarklyApiClient.kt

```diff
@@ -5,11 +5,15 @@ import com.intellij.openapi.extensions.PluginId
 import com.intellij.openapi.project.Project
 import com.launchdarkly.api.ApiClient
 import com.launchdarkly.api.Configuration
+import com.launchdarkly.api.api.AccessTokensApi
 import com.launchdarkly.api.api.FeatureFlagsApi
 import com.launchdarkly.api.api.ProjectsApi
 import com.launchdarkly.api.auth.ApiKeyAuth
+import com.launchdarkly.api.model.Tokens
 import com.launchdarkly.intellij.settings.LaunchDarklyApplicationConfig
 
+private const val APP_URI = "https://app.launchdarkly.com"
```

Comment:
This is usually called the `BASE_URI` and it's also declared [here](https://github.com/launchdarkly/ld-intellij/blob/af9f1a07374b1254ff5460a4c06444f368350a51/src/main/kotlin/com/launchdarkly/intellij/settings/LaunchDarklyApplicationSettings.kt#L67). Maybe we can refactor this to a Constants.kt file so it can be shared in the whole project?

Reply:
Perhaps a better name for this is `DEFAULT_BASE_URI` actually.

Reply:
works for me